qualification/conftest.py

```diff
@@ -511,16 +534,20 @@ async def cbf(
 
 
 @pytest.fixture
-def pass_channels(n_channels: int, narrowband_vlbi: bool) -> slice:
+def pass_channels(
+    band: str, narrowband_decimation: int, narrowband_vlbi: bool, pass_bandwidth: float, n_channels: int
```

Comment:
Why did `n_channels` get moved to last author on this parameter list? pre-commit/`black`?

Reply:
I was messing around with the parameter order to get it similar to `pass_bandwidth` and that's how things ended up. It's purely aesthetic.